README.md

```diff
@@ -1,29 +1,44 @@
 # HTML Build Tools
 
-This repository contains the tools and setup necessary for building the [HTML Standard](https://html.spec.whatwg.org/multipage/) from its [source](https://github.com/whatwg/html).
+This repository contains the tools and instructions necessary for building the [HTML Standard](https://html.spec.whatwg.org/multipage/) from its [source](https://github.com/whatwg/html).
 
-## Setup
+## Prerequisites
 
-1. Get a copy of [wattsi](https://github.com/whatwg/wattsi) and put the resulting binary in your PATH.
-1. Install `wget`.
-1. Check out [the source repo](https://github.com/whatwg/html) into this directory (by checking out into an empty folder then moving the contents here).
+Before building, make sure you have the following commands installed on your system.
 
-Then running build.sh inside this directory will generate the spec, multipage version, entities JSON file, and more.
+- `curl`
+- `egrep`
+- `git`
+- `grep`
+- `perl`
```

Comment:
May also need to mention the perl libraries you need to have installed. My fresh Linux box did not have some XML one. We can leave that for later though. Annoyed at myself I did not write down what I was doing.